crates/ruma-api-macros/src/api/request.rs

```diff
@@ -81,7 +81,9 @@ impl Request {
         let struct_attributes = &self.attributes;
 
         let method = &metadata.method;
-        let path = &metadata.path;
+        let unstable_ts = metadata.unstable_path.as_ref().map(|p| quote! { unstable = #p, });
```

Comment:
What does `ts` stand for? The only thing that comes to mind is "timestamp" 😅

Reply:
TokenStream, though maybe i could change it to `_tokens`